### PR TITLE
add app signing

### PR DESCRIPTION
**Pull request by metaclips**

### CODE REVIEW

**Review by davide-baldo — DISMISSED** [review]

File: .github/actions/build_binaries/action.yml [hunk 1]
```diff
@@ -99,14 +99,23 @@ runs:
           cp target/${{ inputs.target }}/release/libockam_app_lib.d target/release/
 
           # address dependencies for the swiftui app
-          sudo xcode-select --switch /Applications/Xcode_15.0.app/
           brew install create-dmg
 
           if [[ ${{ inputs.target }} == *x86_64* ]]; then
-            make swift_xcode_build ARCH=x86_64
+            make swift_xcode_signed_build \
+              DEVELOPMENT_TEAM="${DEVELOPMENT_TEAM}" \
```
Comment (metaclips) [inline .github/actions/build_binaries/action.yml:106]: Not sure where the `DEVELOPMENT_TEAM` variable defined

Comment (metaclips) [reply .github/actions/build_binaries/action.yml:106]: Same with `PROVISIONING_PROFILE_SPECIFIER`

Comment (davide-baldo) [reply .github/actions/build_binaries/action.yml:106]: it's an environment variable

Comment (metaclips) [reply .github/actions/build_binaries/action.yml:106]: Environment variable are accessible through the vars context https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-variables ${{ vars. DEVELOPMENT_TEAM }}. Let's define it in the `build_release job scope`
```yaml
build_release:
    name: Build Binaries
    needs: create_release
    environment: release
    env:
        DEVELOPMENT_TEAM: ${{ vars.DEVELOPMENT_TEAM }}
```

Comment (davide-baldo) [reply .github/actions/build_binaries/action.yml:106]: done!

**Review by nazmulidris — APPROVED** [review]

**Pull request merged**